packages/aws-cdk/bin/cdk.ts

```diff
@@ -61,6 +61,7 @@ async function parseCommandLineArguments() {
     .command('deploy [STACKS..]', 'Deploys the stack(s) named STACKS into your AWS account', yargs => yargs
       .option('exclusively', { type: 'boolean', alias: 'e', desc: 'only deploy requested stacks, don\'t include dependencies' })
       .option('require-approval', { type: 'string', choices: [RequireApproval.Never, RequireApproval.AnyChange, RequireApproval.Broadening], desc: 'what security-sensitive changes need manual approval' }))
+      .option('ci', { type: 'boolean', desc: 'Force CI detection. Use --no-ci to disable CI autodetection.', default: undefined })
```

Comment:
@rix0rrr The description makes no reference to docker/asset preparation. I thought that this option could be used elsewhere in the future. What about the documentation for this new option? I see that https://github.com/awsdocs/aws-cdk-user-guide/blob/master/doc_source/tools.md is not up to date...

Reply:
That seems good. @Doug-AWS will update the tools section once we release.

Please follow this by something like:

```
if (argv.ci === undefined) {
   argv.ci = process.env.CI !== undefined;
}
```

To make sure that afterwards `argv.ci` is well-defined. Saves having to redo the CI-autodetection logic in every place where it's used. Configuring:

```
default: process.env.CI !== undefined
```

To force the default value to either `true` or `false` might also work.

Reply:
Tools is updated. I should have a new public build of the guide by EOW.